demos/twa-firebase-analytics/src/main/java/com/google/androidbrowserhelper/demos/twa_firebase_analytics/FirebaseAnalyticsLauncherActivity.java

```diff
@@ -36,6 +36,12 @@
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
+        // The Activity was finished before the launch because an existing instance of
+        // the app was brought to front. We don't want to call launchTwa() in this case.
```

Comment:
I find this comment a little confusing - would you be happy with:

super.onCreate may have called finish, in this case, don't do any work.

Reply:
done.